packages/utils/__tests__/__snapshots__/get-asset-list.test.js.snap

```diff
@@ -2842,14 +2842,14 @@ exports[`getAssetLists 1`] = `
         ],
       },
       {
-        "base": "ibc/CF780ADED86E541AD7A8D5B7D33D2E3D9C61D8ECD228B8ED951986D782CB630A",
+        "base": "ibc/8061A06D3BD4D52C4A28FFECF7150D370393AF0BA661C3776C54FF32836C3961",
         "coingecko_id": "pstake-finance",
         "denom_units": [
           {
             "aliases": [
               "ibc/A6E3AF63B3C906416A9AF7A556C59EA4BD50E617EFFE6299B99700CCB780E444",
             ],
-            "denom": "ibc/CF780ADED86E541AD7A8D5B7D33D2E3D9C61D8ECD228B8ED951986D782CB630A",
+            "denom": "ibc/8061A06D3BD4D52C4A28FFECF7150D370393AF0BA661C3776C54FF32836C3961",
```

Comment:
this changed? that's likely not something we want, let's double check this one — this is actually the original reason for the special assets test